main.go

```diff
@@ -86,7 +86,7 @@ func main() {
 	flag.Var(&opts.cookies, "cookie", "Cookie data (alias of -b)")
 	flag.StringVar(&opts.matcherStatus, "mc", "200,204,301,302,307,401,403", "Match HTTP status codes from respose, use \"all\" to match every response code.")
 	flag.StringVar(&opts.matcherSize, "ms", "", "Match HTTP response size")
-	flag.StringVar(&opts.matcherRegexp, "mr", "", "Match regexp")
+	flag.StringVar(&opts.matcherRegexp, "mr", "", "Match regexp, also fuzz keyword allowed")
```

Comment:
I don't think this needs a mention in the help text, as I think the default assumption here is that everything works with the keywords ;)

Reply:
Removed.